telemetry/metrics/gauge.py

```diff
@@ -0,0 +1,69 @@
+from enum import Enum
+from prometheus_client import Gauge as PrometheusGauge
+from typing import Optional
+from nnsight.schema.Request import RequestModel
+from nnsight.schema.Response import ResponseModel
+
+# Labels for the metrics
+request_labels = ('request_id', 'api_key', 'model_key', 'timestamp')
+
+class NDIFGauge:
```

Comment:
Make telemetry a soft link under /src for services that depend on it instead of using Docker volumes..